sormas-api/src/main/resources/strings.properties

```diff
@@ -478,6 +478,7 @@ headingExtendFollowUp = Extend follow-up period
 headingExposureInvestigation = Exposure Investigation
 headingEpiDataSourceCaseContacts = Contacts with Source Case
 headingExposureDetails = Exposure Details
+headingSourceContacts = Source Contacts
```

Comment:
Remove this string (it's already removed on the dev branch) and run I18nConstantGenerator again